ibind/oauth/oauth1a.py

```diff
@@ -19,6 +21,10 @@
     from ibind import IbkrClient
 
 
+STRING_ENCODING = "utf-8"
+INT_BASE = 16
+KEY_VALUE_SEPARATOR = "="
```

Comment:
These variables are private to this file, so to be consistent with the rest of the repo they should start with `_`, eg.

```python
_KEY_VALUE_SEPARATOR = "="
```